RockLib.Configuration.ObjectFactory/.editorconfig

```diff
@@ -0,0 +1,68 @@
+root = true
+
+[*]
+end_of_line = crlf
```

Comment:
Yesterday @NeilSchiefer brought up that we should change this from `crlf` to `lf` to keep everything consistent with what's already in place.